.github/workflows/backport.yml

```diff
@@ -1,30 +1,26 @@
 name: Backport
 on:
-  pull_request_target:
-    types:
-      - closed
-      - labeled
-    paths-ignore:
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
+  issue_comment:
+    types: [created]
+  schedule:
+    # once a day at 13:00 UTC to cleanup old runs
+    - cron: '0 13 * * *'
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: write
 
 jobs:
   backport:
-    name: Backport
-    runs-on: ubuntu-22.04
-    if: >
-      github.event.pull_request.merged &&
-      (
-        github.event.action == 'closed' ||
-        (
-          github.event.action == 'labeled' &&
-          contains(github.event.label.name, 'backport')
-        )
-      )
-    steps:
-      - name: Backport
-        uses: zephyrproject-rtos/action-backport@7e74f601d11eaca577742445e87775b5651a965f #tag=v2.0.3-3
-        with:
-          issue_labels: Backport
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+    uses: dotnet/arcade/.github/workflows/backport-base.yml@aa70f3a7a494063952307a9b1df7ca0ab67bce13
```

Comment:
I pinned the commit out of abundance of caution but we could target `main` if we sufficiently trust Microsoft. This workflow has changed only five times in the last one and a half year.